packages/sources/svelte/src/guidelines/iconography/VtmnIcon/enums.js

```diff
@@ -0,0 +1,9 @@
+export const VARIANT = {
+  DEFAULT: 'default',
+  BRAND: 'brand',
+  REVERSED: 'reversed',
+  POSITIVE: 'positive',
+  INFORMATION: 'information',
+  WARNING: 'warning',
+  NEGATIVE: 'negative',
+};
```

Comment:
Can you please rename this const by "VtmnIconVariant" as we did for badge & tag components enums? Thanks :)

Reply:
I prefer that the const exports are in uppercase, it's ok if I rename by `VTMN_ICON_VARIANT` ?

Reply:
As discussed together, feel free to propose the format `VTMN_ICON_VARIANT` but please do the same for other existing enums in `@vtmn/svelte` stack :)